common/src/point_types.cpp

```diff
@@ -467,4 +467,11 @@ namespace pcl
     return (os);
   }
 
+  std::ostream& 
+  operator << (std::ostream& os, const SCurVSignature210& p)
+  {
+    for (int i = 0; i < 210; ++i)
+    os << (i == 0 ? "(" : "") << p.histogram[i] << (i < 209 ? ", " : ")");
```

Comment:
Move the conditions to be before or after the loop

Reply:
conditions removed from loop